### PR TITLE
#364- coinbase wallet connection

### PR DESCRIPTION
**Pull request by pedrotmr**

### CODE REVIEW

**Comment by sparrowDom** [comment]
Hey could you reintroduce the `contracts/dev.env` file please? Also the dapp doesn't start up right now. Could you look into that as well please and ping me again once you are done: 
Screenshot 2022-02-18 at 19 02 56

**Comment by sparrowDom** [comment]
I don't see `@web3-react/walletlink-connector` under dapp/package.json have you perhaps installed it globally instead of locally? running `yarn add @web3-react/walletlink-connector` in the `dapp` folder should solve the problem

**Comment by pedrotmr** [comment]
Ok, just fixed and pushed. It should be working now

**Comment by sparrowDom** [comment]
Dropping here the original issue for easier reference: https://github.com/OriginProtocol/origin-dollar/issues/634

File: contracts/dev.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PROVIDER_URL=[SET PROVIDER URL HERE]
```
Comment (sparrowDom) [inline contracts/dev.env:1]: nitpick: can you add this text back in

File: dapp/src/components/WalletSelectContent.js [hunk 2]
```diff
@@ -89,23 +96,30 @@ const WalletSelectContent = ({}) => {
...
-              </button>
-            )
-          }
-        )}
+        {[
+          'MetaMask',
+          'Ledger',
+          'CoinbaseWallet',
+          'MyEtherWallet',
+          'WalletConnect',
```
Comment (sparrowDom) [inline dapp/src/components/WalletSelectContent.js:104]: nit: can you change the order of wallets as requested by Micah in the original ticket please: 
`MetaMask, Ledger, Coinbase Wallet, WalletConnect, MEW wallet`

**Review by sparrowDom — COMMENTED** [review]
Left 2 minor comments. Can you PTAL, and ping me once you are done, happy to merge it! Also big thanks for finding the WalletConnect bug and fixing it 👏

File: dapp/src/components/WalletSelectContent.js (hunk 2 above)
Comment (pedrotmr) [reply dapp/src/components/WalletSelectContent.js:103]: Hey, just made those changes! Hope everything is fine now!

Comment (sparrowDom) [reply dapp/src/components/WalletSelectContent.js:103]: Perfect thanks a lot.

**Review by sparrowDom — APPROVED** [review]
Great job on this one @pedrotmr 👏

**Pull request merged**